miden/src/cli/data.rs

```diff
@@ -99,11 +99,29 @@ impl OutputFile {
                 println!("Writing data to output file");
 
                 // write outputs to output file
-                serde_json::to_writer_pretty(file, &Self { outputs })
+                serde_json::to_writer_pretty(
+                    file,
+                    &Self {
+                        outputs: outputs.iter().map(|v| v.to_string()).collect(),
+                    },
+                )
             }
 
             // no path provided - write outputs to stdout
-            None => serde_json::to_writer_pretty(std::io::stdout(), &Self { outputs }),
+            None => {
+                print!(
```

Comment:
nit: let's replace `print` with `println` and get rid of the explicit newline character.